Assets/2_MainScene/S2_Scripts/S2_EnemyScripts/S2_EnemyManager.cs

```diff
@@ -0,0 +1,360 @@
+                        { 
+                            if (difficultyScale >= -1 && difficultyScale <= 1) 
+                                SpawnEnemy(3);
+                            else
+                                EnemySpawnDifficultyAdjuster(difficultyScale); 
+                        }
+                        else
+                            SpawnEnemy(2);
+                    }
+                    else
+                        SpawnEnemy(1);
+                }
+                else
+                    SpawnEnemy(0);
+                break;
+            case "Medium":
+                if (RNG > minion_M[choice] + speeder_M[choice])
+                {
+                        if (RNG > minion_M[choice] + speeder_M[choice] + tank_M[choice])
+                            SpawnEnemy(3);
+                        else
+                            SpawnEnemy(2);
+                }
+                else
+                {
+                    int x = Random.Range(0, 100);
+                    if (x <= 50)
+                        SpawnEnemy(0);
+                    else
+                        SpawnEnemy(1);
+                }
+                break;
+            case "Hard":
+                if (RNG > speeder_H[choice])
+                {
+
+                    if (RNG > speeder_H[choice] + minion_H[choice] + tank_H[choice])
+                        SpawnEnemy(3);
+                    else
+                    {
+                        if (difficultyScale <= -2)
+                        {
+                            int x = Random.Range(0, 100);
+                            if (x < 50) SpawnEnemy(0);
+                            else SpawnEnemy(2);
+                        }
+                        else
+                        {
+                            if (RNG > speeder_H[choice] + minion_H[choice])
+                                SpawnEnemy(2);
+                            else
+                                SpawnEnemy(0);
+                        }    
+                    }
+                }
+                else
+                    SpawnEnemy(1);
+                break;
+            case "Very Hard":
+                if (RNG > minion_VH[choice])
+                {
+                    if (RNG > (minion_VH[choice] + speeder_VH[choice]))
+                    {
+
+                        if (RNG > minion_VH[choice] + speeder_VH[choice] + tank_VH[choice])
+                        {
+                                SpawnEnemy(3);
+                        }
+                        else
+                            SpawnEnemy(2);
+                    }
+                    else
+                        SpawnEnemy(1);
+                }
+                else
+                    SpawnEnemy(0);
+                break;
+            default:
+                break;
+        }
+    }
+
+    public void RemoveFromEnemiesInWave(S2_EnemyStats x)
+    {
+        enemiesInWave.Remove(x);
+    }
+
+
+    void Start()
```

Comment:
Position in script? 
Also Instance should be in an awake function or it may not be set fast enough for script B to find it from it's own Start function